pkgs/by-name/gi/gibo/package.nix

```diff
@@ -1,40 +1,50 @@
 {
   lib,
-  stdenv,
+  buildGoModule,
   fetchFromGitHub,
-  coreutils,
-  findutils,
-  git,
+  installShellFiles,
 }:
-
-stdenv.mkDerivation rec {
+buildGoModule (finalAttrs: {
   pname = "gibo";
-  version = "1.0.6";
+  version = "3.0.14";
 
   src = fetchFromGitHub {
     owner = "simonwhitaker";
     repo = "gibo";
-    rev = version;
-    sha256 = "07j3sv9ar9l074krajw8nfmsfmdp836irsbd053dbqk2v880gfm6";
+    tag = "v${finalAttrs.version}";
+    sha256 = "sha256-6w+qhwOHkfKt0hgKO98L6Si0RNJN+CXOOFzGlvxFjcA=";
   };
 
-  installPhase = ''
-    mkdir -p $out/bin $out/share/bash-completion/completions
-    cp gibo $out/bin
-    cp gibo-completion.bash $out/share/bash-completion/completions
+  vendorHash = "sha256-pD+7yvBydg1+BQFP0G8rRYTCO//Wg/6pzY19DLs42Gk=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/simonwhitaker/gibo/cmd.version=v${finalAttrs.version}"
+  ];
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  postInstall = ''
+    installShellCompletion --cmd gibo \
+      --bash <($out/bin/gibo completion bash) \
+      --fish <($out/bin/gibo completion fish) \
+      --zsh <($out/bin/gibo completion zsh)
+  '';
 
-    sed -e 's|\<git |${git}/bin/git |g' \
-        -e 's|\<basename |${coreutils}/bin/basename |g' \
-        -i "$out/bin/gibo"
-    sed -e 's|\<find |${findutils}/bin/find |g' \
-        -i "$out/share/bash-completion/completions/gibo-completion.bash"
+  installCheckPhase = ''
+    runHook preInstallCheck
+    $out/bin/gibo version | grep -F "${finalAttrs.version}"
+    runHook postInstallCheck
   '';
 
   meta = {
     homepage = "https://github.com/simonwhitaker/gibo";
-    license = lib.licenses.publicDomain;
+    license = lib.licenses.unlicense;
     description = "Shell script for easily accessing gitignore boilerplates";
     platforms = lib.platforms.unix;
     mainProgram = "gibo";
```

Comment:
The 1.0.6 version is *ancient* (Oct 2017). You are likely the first person to care about this package again. You could consider adding yourself to maintainers, as well as adding an update script (`passthru.updateScript = nix-update-script { };`) to make sure this package won't be forgotten about again. Of course, there is no obligation to commit to being a maintainer, just something to consider.

Reply:
formatter is complaining (needs to be `{ }`, not `{}`), but that does help already.